Cargo.toml

```diff
@@ -24,8 +24,7 @@ latest_rust = []  # use latest Rust features (otherwise, support Rust 1.34)
 [dependencies]
 base64 = "0.10"
 bincode = "1.0"
-bitcoin = { version = "0.23", features = ["use-serde"] }
-bitcoin_hashes = "0.7.6"
+bitcoin = { git = "https://github.com/kallewoof/rust-bitcoin.git", branch = "2019-07-signet", features = ["use-serde"] }
```

Comment:
Rust-bitcoin has merged signet support so this should use the upstream version. See https://github.com/rust-bitcoin/rust-bitcoin/pull/291

Reply:
I realized this was cherry picked in a different PR, ignore above.